docs/installation-and-operation/serialization.md

```diff
@@ -296,8 +296,152 @@ If you've written scripts to automate serialization, you'll need to:
 
 Before exporting, you can also run a Metabase command to [drop entity IDs](./commands.md#drop-entity-ids).
 
+## Serializing Metabase via the API
+
+> Just like the CLI serialization commands, these endpoints are only available for [Pro](https://www.metabase.com/product/pro) and [Enterprise](https://www.metabase.com/product/enterprise) plans.
+
+You can initiate imports and exports of serialized Metabase data via the API.
+
+There are two endpoints:
+
+- `/api/ee/serialization/export`
+- `/api/ee/serialization/import`
+
+For now, these endpoints are synchronous. If the serialization process takes too long, the request can time out. In this case, we suggest using the CLI commands.
+
+### Use parameters to customize what Metabase exports
+
+You can append optional parameters to tell Metabase what to include or exclude from the export. You can also combine parameters (excluding, of course, `all_collections` and selective collections).
+
+So, assuming you're testing on `localhost`, and you want to exclude all collections from the export, you'd format the URL like so:
+
+```html
+http://localhost:3000/api/ee/serialization/export?all_collections=false
+```
+
+You can include multiple parameters, separated by `&`. For example, to exclude both the settings and the data model from the export:
+
+```html
+http://localhost:3000/api/ee/serialization/export?data_model=false&settings=false
+```
+
+### Example params
```

Comment:
For these params, I think it is worth mentioning the default values.

For the diretory name, the default diretory name is `metabase-yyyy-mm-dd_HH-mm`